certificate-service/pdfbox-generator/src/test/java/se/inera/intyg/certificateservice/pdfboxgenerator/pdf/value/PdfElementValueGeneratorTest.java

```diff
@@ -0,0 +1,119 @@
+package se.inera.intyg.certificateservice.pdfboxgenerator.pdf.value;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificate.FK7210_CERTIFICATE;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificate.FK7472_CERTIFICATE;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7210_PDF_FODELSEDATUM_FIELD_ID;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7210_QUESTION_BERAKNAT_FODELSEDATUM_ID;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7472_PDF_PERIOD_FIELD_ID_PREFIX;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7472_PDF_SYMPTOM_FIELD_ID;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7472_QUESTION_PERIOD_ID;
+import static se.inera.intyg.certificateservice.domain.testdata.TestDataCertificateModelConstants.FK7472_QUESTION_SYMPTOM_ID;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.certificateservice.domain.certificatemodel.model.PdfValueType;
+
+@ExtendWith(MockitoExtension.class)
+class PdfElementValueGeneratorTest {
```

Comment:
I think the tests you have added are great, and clearly verifies that correct valueConverter is called. But im missing a test that verifies that we are actually getting a List<PdfField> returned.

Could you please add that aswell? If we changed the logic now so that this service returns a list of another object, the tests would not catch that!